api/net/Program.cs

```diff
@@ -258,4 +269,6 @@
 
 app.MapControllers();
 
+app.MapHub<WorkOrderHub>("/api/work-order-hub");
```

Comment:
This should be based on configuration and default to this value.

Reply:
Updated, thanks